c2corg_api/models/user.py

```diff
@@ -172,3 +173,24 @@ def validate_password(self, plain_password):
             'validator': colander.OneOf(default_langs)
         }
     })
+
+# Make sure that user email changes are propagated to mailing lists as well
+trigger_ddl = DDL("""
+CREATE OR REPLACE FUNCTION users.update_mailinglists_email() RETURNS TRIGGER AS
+$BODY$
+BEGIN
+  UPDATE """ + sympa_schema + """.subscriber_table
+  SET user_subscriber = NEW.email
+  WHERE user_subscriber = OLD.email;
```

Comment:
OK I can change this. I guess it must be consistent with the primary key definition: use ``email`` or ``user_id`` depending what one is used in the primary key?

By the way I have just tested that the email is actually updated in the ML table when changing it in ``users.user``.